src/Common/src/CoreLib/System/Runtime/InteropServices/IDispatch.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices.ComTypes;
+using System.Text;
+
+namespace System.Runtime.InteropServices
+{
+    [ComImport]
+    [Guid("00020400-0000-0000-C000-000000000046")]
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    internal interface IDispatch
+    {
+        void GetIDsOfNames(
+            ref Guid riid,
+            [MarshalAs(UnmanagedType.LPArray, ArraySubType = UnmanagedType.LPWStr, SizeParamIndex = 2), In]
+            string[] rgszNames,
+            int cNames,
+            int lcid,
+            [Out] int[] rgDispId);
+
+        ITypeInfo GetTypeInfo(
+            int iTInfo,
+            int lcid);
+
+        int GetTypeInfoCount();
+
+        void Invoke(
+            int dispIdMember,
+            ref Guid riid,
+            int lcid,
+            InvokeFlags wFlags,
+            ref DISPPARAMS pDispParams,
+            out object pVarResult,
+            IntPtr pExcepInfo,
+            IntPtr puArgErr);
+    }
+
+    [Flags]
+    internal enum InvokeFlags : short
+    {
+        Method = 1,
```

Comment:
From https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md#naming:

Interop signatures / structs / constants should be defined using the same name / capitalization / etc. that's used in the corresponding native code.

So this should be `DISPATCH_METHOD` and so on.